itest/lib/brim-selectors.ts

```diff
@@ -0,0 +1,14 @@
+import * as locators from "src/js/test/locators"
+import {quote} from "./utils"
+
+export default class BrimSelectors {
+  toolbarView = `button[aria-label="View"]`
+  detailPane = locators.detailPane.css
+  detailPaneSections = this.detailPane + " h4"
+
+  viewerCellContaining(text) {
+    return (
+      locators.viewerResults.xpath + `//span[contains(text(), ${quote(text)})]`
+    )
+  }
+}
```

Comment:
I will add all of the selectors in `src/js/test/integration.ts` and probably all the locators in `src/js/test/locators` into this file if this design gets approval.